### PR TITLE
03-995 Add clinic metrics view

### PR DESCRIPTION
**Pull request by makombe**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
Selection_013

*None.*

## Related Issue

[https://issues.openmrs.org/browse/O3-995](https://issues.openmrs.org/browse/O3-995)

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  return (
+    <>
+      <MetricsHeader />
+      <div className={styles.clinicMetricsContainer}>
+        <MetricsCard
+          totalsLabel={t('patients', 'Patients')}
+          totalsValue={metrics ? metrics.scheduled_appointments : 0}
+          headerLabel={t('scheduledAppointment', 'Scheduled appts. today')}
```
Comment (donaldkibet) [inline packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx:39]:
```suggestion
          label={t('patients', 'Patients')}
          value={metrics ? metrics.scheduled_appointments : 0}
          title={t('scheduledAppointment', 'Scheduled appts. today')}
```

**Review by donaldkibet — APPROVED** [review]
LGTM 🚀  Thanks @makombe

File: packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx (hunk 1 above)
Comment (denniskigen) [reply packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx:39]: Use `scheduledAppointments` instead.

File: packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        />
+        <MetricsCard
+          totalsLabel={t('patients', 'Patients')}
+          totalsValue={metrics ? metrics.patients_waiting_for_service : 0}
+          headerLabel={t('waitingFor', 'Waiting for:')}
+          childComponent={
+            <Dropdown
+              style={{ marginTop: '1.5rem' }}
+              id="inline"
+              label="Triage"
```
Comment (denniskigen) [inline packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx:49]: label={t('triage', 'Triage')}

File: packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      id: 'option-1',
+      text: 'Triage',
+    },
+    {
+      id: 'option-2',
+      text: 'Adult return',
+    },
+    {
+      id: 'option-3',
+      text: 'Pharamacy',
```
Comment (denniskigen) [inline packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx:25]: Small typo here.

Comment (denniskigen) [inline packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx:25]:
```suggestion
      text: 'Pharmacy',
```

**Review by denniskigen — APPROVED** [review]
LGTM sans some minor changes related to translations. Nice work, @makombe!

**Comment by denniskigen** [comment]
Also, one thing, @donaldkibet and @makombe. When metrics are unavailable, should we use 0 or `--` as a placeholder value?

File: packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx (hunk 1 above)
Comment (makombe) [reply packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx:39]: Thanks @denniskigen  and @donaldkibet  for you reviews let me address the suggested changes

**Pull request merged**